windows/lcow_sandbox.go

```diff
@@ -0,0 +1,521 @@
+// +build windows
+
+package windows
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"sync"
+	"syscall"
+	"time"
+	"unsafe"
+
+	"golang.org/x/sys/windows"
+
+	winio "github.com/Microsoft/go-winio"
+	"github.com/Microsoft/hcsshim"
+	"github.com/Microsoft/opengcs/client"
+	"github.com/docker/swarmkit/log"
```

Comment:
wrong import here, should be `github.com/containerd/containerd/log`. goimports does this to me all the time

Reply:
Oops. Good catch.